tools/constraints-bionic.txt

```diff
@@ -2,6 +2,7 @@ attrs==17.4
 flake8==3.5.0
 py==1.5.2
 pycodestyle==2.3.1
+pyflakes==2.1.1
```

Comment:
Why not the latest version 2.2.0?

Reply:
We could, and probably should, I'll bump this to 2.2.0